pkg/monitor/monitor/util.go

```diff
@@ -638,6 +638,21 @@ func getMonitorService(monitor *v1alpha1.TidbMonitor) []*core.Service {
 	var services []*core.Service
 	monitorLabel := label.New().Instance(monitor.Name).Monitor()
 	labels := label.NewMonitor().Instance(monitor.Name).Monitor()
+
+	reloaderPortName := "tcp-reloader"
+	prometheusPortName := "http-prometheus"
+	grafanaPortName := "http-grafana"
+
+	if monitor.Spec.Reloader.Service.PortName != nil {
```

Comment:
I recommend add an access method for each Spec to get the port name to encapsulate the details of port name calculation.

Reply:
updated.